src/contents/articles/public_coding_30_may_2015.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Public coding - Sat May 30, 2015
+author: 'Andre'
+date: 2015-05-29
+template: article.jade
+---
+
+Namespace will be hosting a public coding event this Saturday, the 30th of May, 2015. 
```

Comment:
Re-phrase this to say something like : 'We will be coding in public'. Saying we are hosting sounds like we are throwing an event with official permission.